extensions/ql-vscode/src/extension.ts

```diff
@@ -775,6 +774,14 @@ async function activateWithInstalledDistribution(
       cancellable: true
     })
   );
+
+  ctx.subscriptions.push(
+    commandRunner('codeQL.monitorRemoteQuery', async (
```

Comment:
`commandRunner` does not supply a progress monitor. Use this instead:

```suggestion
    commandRunnerWithProgress('codeQL.monitorRemoteQuery', async (
```

Reply:
Do we need this even if we're not using the progress API?

Reply:
No, we don't. You can remove it.